### PR TITLE
CrowdStrikeFalcon - Add logging for `refresh stream` requests

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def collect_verticles_from_detection(self, detection_id: str) -> Generator[tuple
                 error,
                 message=(
                     f"Failed to collect verticles for detection {detection_id}: "
-                    "{error.response.status_code} {error.response.reason}"
+                    f"{error.response.status_code} {error.response.reason}"
```
Comment (lvoloshyn-sekoia) [inline CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py:125]: This is not related, just noticed an issue

**Review by squioc — APPROVED** [review]
LGTM
